tests/tantivy_kapiche_test.py

```diff
@@ -513,7 +513,7 @@ def test_create_readers(self):
 class TestSearcher(object):
     def test_searcher_repr(self, ram_index, ram_index_numeric_fields):
         assert ram_index.searcher().num_docs == 3
-        assert ram_index.searcher().num_segments == 2
+        assert ram_index.searcher().num_segments == 1
```

Comment:
why does this change?

Reply:
I increased the memory allocation on the writer. This was an intentional change.